recipes/mgis/conda_build_config.yaml

```diff
@@ -0,0 +1,24 @@
+c_compiler:
```

Comment:
Please remove everything except the versions. Still: Do you need such an old compiler?

Reply:
That's also a good question.

The reason for picking such an old version (8) of the gcc compilers is because [Code_Aster](https://github.com/conda-forge/code-aster-feedstock) is also very picky wrt. gcc compiler versions. My assumption was that it would be best to align the dependencies of Code_Aster to the same compiler. 

But I am not certain that is necessarily required. So I will probably try to loosen this restrictions after some trial and error

Reply:
I have removed everything except the compiler versions. I'll keep the old versions for now, and I will try to loosen the version pinning once I'm sure that the dependencies are unaffected by using a more recent compiler version.

Hope it's okay that I consider this conversation as resolved.